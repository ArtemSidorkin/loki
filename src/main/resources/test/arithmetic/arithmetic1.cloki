println(10 + 20)
println(10 + 20 + 30)
println(10 + 20 + 30 + 40)
println()

println(20 - 10)
println(30 - 20 - 10)
println(30 - 20 - 10 - 40)
println()

println(10 * 20)
println(10 * 20 * 30)
println(10 * 20 * 30 * 40)
println()

println(20 / 10)
println(30 / 20 / 10)
println(30 / 20 / 10 / 40)

