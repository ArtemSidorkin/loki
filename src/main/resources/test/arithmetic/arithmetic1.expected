30
60
100

10
0
-40

200
6000
240000

2
0.15
0.00375